### PR TITLE
Suppress Ruby 2.7's warning about keyword arguments

### PR DESCRIPTION
**Pull request by oieioi**
Suppress Ruby 2.7's warning about keyword arguments. I think the version of capybara should be upgraded as well.

cf. 
- https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/
-  https://github.com/teamcapybara/capybara/commit/a76a35fbc8bb5baceb13c50ede1b9ee551f674d7

### CODE REVIEW

**Comment by luke-hill** [comment]
Could you do 2 things.
- Add a note in the Changelog, try following the format done in the most recent updates (I'm aware it's not fully future proofed)
- Fix the rubocop issue by updating the todo list.

**Comment by luke-hill** [comment]
Also, Capybara has an issue we've yet to fully diagnose. I am coming into some free time later this week where I'm going to tackle a lot of outstanding issues and cut 1/2 releases of the gem. So I'll look into this.

**Comment by oieioi** [comment]
@luke-hill 
Thank you for the review.

> Could you do 2 things.
> - Add a note in the Changelog, try following the format done in the most recent updates (I'm aware it's not fully future proofed)
> - Fix the rubocop issue by updating the todo list.

I did that and the build passed.

[In this build, the old version of capybara is used, so there are warnings](https://travis-ci.com/github/site-prism/site_prism/jobs/332488066). This can be suppressed by using [3.31.0 or higher](https://github.com/teamcapybara/capybara/commit/a76a35fbc8bb5baceb13c50ede1b9ee551f674d7).

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 ### Changed
 
 ### Fixed
+- Fixed warnings about keyword arguments in Ruby 2.7
+  - The official explanation of keyword arguments in Ruby 2.7 can be found [HERE](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/)
+([oieioi])
```
Comment (luke-hill) [inline CHANGELOG.md:11]: Add this below all other contributors at bottom and I'll merge. It just makes formatting proper.

Comment (oieioi) [reply CHANGELOG.md:11]: @luke-hill Added.

**Pull request merged**